ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -129,7 +129,8 @@ private static void validateType(
       Schema.Type.FLOAT32,
       Schema.Type.FLOAT64,
       Schema.Type.BOOLEAN,
-      Schema.Type.STRING
+      Schema.Type.STRING,
+      Schema.Type.STRUCT
```

Comment:
This issue was created for STRUCT->STRING. But the issue also happens for ARRAY->STRING. Look at the `KsqlJsonDeserializer` class and `processString` method. That's where the old JSON_SR deserializer was doing.

Do you think you can add the ARRAY conversion to this PR or would you need another one?